src/Files/UserControls/StatusBarControl.xaml

```diff
@@ -27,52 +27,62 @@
 
     <Grid
         Height="32"
+        Padding="8,0"
         HorizontalAlignment="Stretch"
-        Background="{ThemeResource StatusBarBackgroundBrush}">
+        VerticalAlignment="Stretch"
+        Background="{ThemeResource StatusBarBackgroundBrush}"
+        ColumnSpacing="12">
         <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="Auto" />
+            <ColumnDefinition Width="Auto" />
+            <ColumnDefinition Width="Auto" />
             <ColumnDefinition Width="*" />
             <ColumnDefinition Width="Auto" />
         </Grid.ColumnDefinitions>
 
+        <TextBlock
+            x:Name="DirectoryItemCount"
+            Grid.Column="0"
+            VerticalAlignment="Center"
+            x:Load="{x:Bind ShowInfoText, Mode=OneWay}"
+            Text="{x:Bind DirectoryPropertiesViewModel.DirectoryItemCount, Mode=OneWay}" />
+        <TextBlock
+            x:Name="SelectedItemsCountString"
+            Grid.Column="1"
+            VerticalAlignment="Center"
+            x:Load="{x:Bind ShowInfoText, Mode=OneWay}"
+            Text="{x:Bind SelectedItemsPropertiesViewModel.SelectedItemsCountString, Mode=OneWay}"
+            Visibility="{x:Bind SelectedItemsPropertiesViewModel.IsItemSelected, Mode=OneWay}" />
+        <TextBlock
+            x:Name="ItemSize"
+            Grid.Column="2"
+            VerticalAlignment="Center"
+            x:Load="{x:Bind ShowInfoText, Mode=OneWay}"
+            Text="{x:Bind SelectedItemsPropertiesViewModel.ItemSize, Mode=OneWay}"
+            Visibility="{x:Bind SelectedItemsPropertiesViewModel.IsItemSelected, Mode=OneWay}" />
+
+        <Button
+            x:Name="FullTrustStatus"
+            Grid.Column="4"
```

Comment:
Why not `Grid.Column="3"`?

Reply:
Column 3 is to provide blank space so that the button column is on the far right at all times.

Reply:
Why not HorizontalAlignment="Right"?

Reply:
That would work but this would be a bit more flexible if we plan to add more controls to the right column in the future.